R/internal.R

```diff
@@ -69,6 +70,12 @@
     is_valid_shape
 }
 
+.is_valid_lineTypes <- function(linetype) {
+    valid_linetypes <- c("solid", "dotted", "dashed", "dotdash", "longdash", "twodash")
+    is_valid_linetype <- ifelse(linetype %in% valid_linetypes, TRUE, FALSE)
```

Comment:
return `ifelse(linetype %in% valid_linetypes, TRUE, FALSE)` output directly, no need for this extra parameter

Reply:
This function will go away - as we are no longer going to support aesthetic related parameters. I will leave everything in internal.R as is for now.